login/FrontQuery.js

```diff
@@ -0,0 +1,7 @@
+class FrontQuery{
+    constructor(login,password,name,surname,id,rankId,date){
```

Comment:
Maybe the constructor doesn't need parameters it doesn't use?

Reply:
we need it in future

Reply:
I suggest you to read about YAGNI, long story short, don't do what you don't need yet.

Reply:
oh,sry,i am tired and read wrong part of code,i fixed it
thx for comment